core/spec.md

```diff
@@ -4168,6 +4184,50 @@ and the following Resource level attributes:
     MUST be `external`.
   - If present, MUST be non-empty.
 
+#### `deprecated`
+
+- Type: Object containing the following properties:
+  - `effective`<br>
+    An OPTIONAL property indicating the time when the Resource entered, or will
+    enter, a deprecated state. The date MAY be in the past or future. If this
+    property is not present the Resource is already in a deprecated state.
+    If present, this MUST be an [RFC3339][rfc3339] timestamp.
+
+  - `removal`<br>
+    An OPTIONAL property indicating the time when the Resource MAY be removed.
+    The Resource MUST NOT be removed before this time. If this property is not
+    present then client can not make any assumption as to when the Resource
+    might be removed. Note: as with most properties, this property is mutable.
+    If present, this MUST be an [RFC3339][rfc3339] timestamp and MUST NOT be
+    sooner than the `effective` time if present.
+
+  - `alternative`<br>
+    An OPTIONAL property specifying the URL to an alternative Resource the
+    client can consider as a replacement for this Resource. There is no
+    guarantee that the referenced Resource is an exact replacement, rather the
+    client is expected to investigate the Resource to determine if it is
+    appropriate.
+
+  - `docs`<br>
+    An OPTIONAL property specifying the URL to additional information about
+    the deprecation of the Resource. This specification does not mandate any
+    particular format or information, however some possibilities include:
+    reasons for the deprecation or additional information about likely
+    alternative Resources. The URL MUST support an HTTP GET request.
+
+  Note that an implementation is not mandated to use this attribute in
+  advance of removing a Resource, but is it RECOMMENDED that they do so.
+- Constraints:
+  - OPTIONAL
+- Examples:
+  - `"deprecated": {}`
+  - ```
+    "deprecated": {
+      "removal": "2030-12-19T00:00:00-00:00",
```

Comment:
Eek, let's not use an offset of -00:00 - that indicates an "unknown local offset"; that's unlikely to be usefully represented in most software, and I doubt that many users know what it means anyway. I would strongly recommend using Z here (and everywhere).

Reply:
Fixed here and in endpoint's spec.
I'll work on add the "Z" in next week's PR as that's a bit of a bigger change.